release-notifier/internal/mail/mail.go

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ******************************************************************************/
+
+package mail
+
+import (
+	"bytes"
+	"fmt"
+	"net/smtp"
+	"os"
+	"text/template"
+)
+
+const smtpServer = "smtp.gmail.com"
+const smtpPort = "587"
+const mailTemplate = "templates/mail.html.tmpl"
+const recipentMail = "TRACTUSX_MAILINGLIST"
```

Comment:
I would follow the pattern of the other constant names holding environment variable names and add the Env suffix

Reply:
Renamed.